controllers/dashboard-routes.js

```diff
@@ -19,18 +19,40 @@ router.get('/', withAuth, (req,res) => {
         attributes: ['id','user_id','service_id','is_active','auto_renewal_date'],
         include: {
             model: Service,
-            attributes: ['name','cost','cost_basis']
+            attributes: ['name', 'cost', 'cost_basis']
         }
     })
         .then(dbSubData => {
+            console.log(dbSubData);
```

Comment:
probably ought to remove the console.logs as well

Reply:
sure